AsyncDisplayKit/ASCollectionViewLayoutFacilitatorProtocol.h

```diff
@@ -0,0 +1,19 @@
+//
+//  ASCollectionViewLayoutFacilitatorProtocol.h
+//  Pods
+//
+//  Created by Bin Liu on 1/13/16.
```

Comment:
Can you use the standard comment at the top of files?  It's supposed to call out the copyright unfortunately :(